setup.py

```diff
@@ -0,0 +1,91 @@
+import os
+
+from setuptools import find_packages  # , setup
+from numpy.distutils.core import setup, Extension
+from numpy.distutils.fcompiler import get_default_fcompiler, CompilerNotFound
+
+here = os.path.dirname(__file__)
+with open(os.path.join(here, "README.md"), encoding="utf-8") as f:
+    long_description = f.read()
+
+install_requires = [
+    "numpy",
+]
+
+# figure out which compiler we're going to use
+compiler = get_default_fcompiler()
+# set some fortran compiler-dependent flags
+f90flags = []
+if compiler == "gnu95":
+    # f90flags.append('-fno-range-check')
+    # f90flags.append('-ffree-form')
+    # These are the flags I used in the manual version (might have some more here)
+    f90flags.append("-fdefault-real-8")
+    f90flags.append("-ffree-line-length-200")
+elif compiler == "intel" or compiler == "intelem":
+    f90flags.append("-132")
+#  Set aggressive optimization level
+f90flags.append("-O3")
+#  Suppress all compiler warnings (avoid huge CI log files)
+f90flags.append("-w")
+
+# for this API we will only expose a single extension?
+ext_modules = [
+    Extension(
+        name="mod_aerobulk_wrap",  # this somehow needs to match the mod_aerobulk_wrap.* files. Ideally I would like to name this just aerobulk (for later.)
```

Comment:
Feel free to rename it however you want. This naming was just my first attempt.

Reply:
Left that to #5 for later!